docs/components/footer.css

```diff
@@ -0,0 +1,20 @@
+.sitemapMasonry {
+  columns: 3 150px;
+  column-gap: 20px;
+  margin: 10px 0px 10px;
+}
+
+.sitemapMasonry > div {
+  display: inline-block;
+  width: 100%;
+  padding: 10px 0px;
+}
+
+.sitemapMasonry > div div {
```

Comment:
wat

Reply:
To clarify, I think you're selecting all divs within divs that are direct children of `sitemapMasonry`?

Styling like this is often quite brittle and prone to unexpected side effects (any divs within divs in this layout will have padding applied, often that will be wrong)

I would advise a more direct style on the class if possible